target/linux/mvebu/cortexa9_neon/base-files/lib/upgrade/linksys.sh

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2014-2015 OpenWrt.org
+#
+
+linksys_get_target_firmware() {
```

Comment:
For this file, I think it might be better to move it to target base-files (target/linux/mvebu/base-files/lib/upgrade/linksys.sh) instead of having two copies. The size impact will be tolerable in comparison to the disadvantage of having this twice.